src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporterHelperExtensions.cs

```diff
@@ -0,0 +1,82 @@
+// <copyright file="OtlpLogExporterHelperExtensions.cs" company="OpenTelemetry Authors">
```

Comment:
I would have expected this to just be a file move rather than a copy. I think we can delete the OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs directory with this PR. Any reason to keep it?

Reply:
No reason. Wanted to keep the delete part separate

Reply:
Need to figure out how to merge the changelog in to this one.

Reply:
The changelog is pretty sparse - probably didn't really make sense to have its own changelog. I think it would be fine to just merge in the messages into the corresponding release of the main changelog.

Reply:
> No reason. Wanted to keep the delete part separate

It's not a huge deal in this case, but normally I try not to separate a move with a copy then delete. I don't look at git history too often but it's nice when I need it.

Reply:
Removed :)